test/integration/lib/integration-server.ts

```diff
@@ -54,9 +54,7 @@ export class IntegrationServer {
       }
     })
 
-    this.server = this.app.listen(port, () => {
-      console.log(`Integration server listening on port ${port}`)
```

Comment:
I removed this log as it was unnecessary but it got me thinking about pino here. Frankly I don't think it makes sense to use a structured logger for something that is exclusively going to be viewed from a development/github actions context. Wouldn't be consumed by a log aggregator and just makes it harder to read.